akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -162,7 +162,8 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
   protected def onPersistRejected(cause: Throwable, event: Any, seqNr: Long): Unit = {
     log.warning(
       "Rejected to persist event type [{}] with sequence number [{}] for persistenceId [{}] due to [{}].",
-      event.getClass.getName, seqNr, persistenceId, cause.getMessage)
+      event.getClass.getName, seqNr, persistenceId,
+      s"[${cause.getMessage}] [${Logging.stackTraceFor(cause)}]")
```

Comment:
[] around the stack trace does not really make sense I think, should it be after an \n instead?

Reply:
@ktoso you're absolutely right. Fixed.